test/rb/test_rb.py

```diff
@@ -726,6 +729,58 @@ def test_rb(self, nq, pattern_type, multiplier_type):
             self.assertEqual(circ_index, len(rb_purity_circs),
                              "Error: additional purity circuits exist")
 
+    def test_interleavd_rb(self):
+        """Further tests for interleaved RB."""
+        # The interleaved RB sequences will not be executed
+        # only to check the generator function doesn't error
+
+        # 1Q Clifford gates
+        for gate in [HGate(), SGate(), SdgGate(), XGate(), YGate(), ZGate()]:
+            _, _, _ = rb.randomized_benchmarking_seq(
+                1, [5], rb_pattern=[[0]], interleaved_elem=[gate])
```

Comment:
This needs to assert something on the output. A good test will not just check that there isn't a failure when calling a method, but also that the output meets the expected result. I'm assuming the output here is randomized so doing a straight equals assertion isn't easy (although ideally in this case the function would take a `seed` parameter to enable a reproducible result you can then use for testing) in which case you should assert the types, shapes, etc. If it's deterministic and returns the same result for the same input then just assert on that. This applies to all the places where you're calling `randomized_benchmarking_seq` and just ignoring the result.

Reply:
thanks for the feedback - I added assertions to the tests.